openapi.yaml

```diff
@@ -143,374 +82,417 @@ paths:
         - $ref: '#/parameters/phoneType'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - name: primary
+          in: query
+          type: string
+          required: false
+          description: Filter on primary or non-primary phone numbers
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PhonesResultObject'
-        400:
-          description: Bad request
-          schema:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
+            $ref: '#/definitions/PhoneSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
     post:
-      description: Create phone numbers for a person
+      description: Create phone number for a person
       tags:
         - phones
       operationId: createPhones
       parameters:
-        - name: phoneBody
+        - name: body
           in: body
-          description: Post body for the new Phone
+          description: Post body for the new phone
           required: true
           schema:
-            $ref: '#/definitions/NewPhoneResultObject'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+            $ref: '#/definitions/PhonePostBody'
+        - $ref: '#/parameters/osuId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/PhoneResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/emails:
+    get:
+      description: Get emails for a person
+      tags:
+        - emails
+      operationId: getEmails
+      parameters:
+        - $ref: '#/parameters/osuId'
+        - name: emailType
+          in: query
+          type: string
+          required: false
+          enum: &emailTypeEnums
+            - ONID
+            - STU
+            - VEND
+            - VWEB
+            - EMPL
+            - VPAY
+            - EMER
+          description: |
+            Type of email
+            - `STU` - Non-directory Student
+            - `ONID` - Student
+            - `VEND` - Vendor
+            - `VWEB` - Vendor
+            - `EMPL` - Employee
+            - `VPAY` - Vendor Pay
+            - `EMER` - Emergency Contact Persons Email for Student
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PhoneResultObject'
-        400:
-          description: Bad request
+            $ref: '#/definitions/EmailSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+    post:
```

Comment:
Let's remove this method first. Since this method will insert new data (not supported in v1) into the database, we need to get approval from Joe or Kent before implement it.